org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/scan/DeviceInformation.java

```diff
@@ -444,4 +463,12 @@ public void setMonitorRole(MonitorRole monitorRole) {
 		this.monitorRole = monitorRole;
 	}
 
+	public Map<String, IDeviceAttribute<?>> getAttributes() {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)